bin/release-dev.sh

```diff
@@ -3,11 +3,10 @@
 set -e
 
 # use UTC timestamp as version
-ver=$(date -u +%Y%m%d%H%M%S)
+timestamp=$(date -u +%Y%m%d%H%M%S)
 
-sed -i -r "s/^__version__ = \"(.*\.dev)\"/__version__ = \"\1${ver}\"/" localstack/__init__.py
+echo "$timestamp" >> VERSION
```

Comment:
if i run the script several times we would keep appending the timestamp, right? not that it's intended to be used that way, but perhaps making it more robust this way?

Reply:
You're right. Will fix that shortly!